integrations/adp/nango.yaml

```
@@ -0,0 +1,74 @@
+        type: HOME | WORK
+
+    StandardEmployee:
+        id: string
+        firstName: string
+        lastName: string
+        email: string
+        displayName: string
+        employeeNumber?: string
+        title: string
+        department:
+        id: string
```

YAMLlint (1.37.1)

[error] duplication of key "id" in mapping

(key-duplicates)